diff --git a/activity/activity.info b/activity/activity.info
index 265fce1..496b0f5 100644
--- a/activity/activity.info
+++ b/activity/activity.info
@@ -1,13 +1,23 @@
 [Activity]
 name = FotoToon
-bundle_id = org.eq.FotoToon
+bundle_id = org.sugarlabs.FotoToon
 icon = historieta
 exec = sugar-activity3 historietaactivity.HistorietaActivity
 show_launcher = yes
 activity_version = 26
+release_date = 2021-02-08
+host_version = 2
 mime_types = application/x-fototoon-activity;
-summary = Interested in telling a story? Easy! Use your pictures and your imagination to create your own comics!
 categories = media language art
 repository = https://github.com/sugarlabs/fototoon-activity.git
+url = https://help.sugarlabs.org/en/fototoon.html
 max_participants = 1
 license = GPLv3+
+metadata_license = CC0-1.0
+summary = Create your own comic
+description = FotoToon allows you to create everything from a simple “comic” strip to a long elaborate illustrated document. You will be able to import many kinds of images into the panes of your project and add text bubbles to help tell the story.
+tags = Graphics
+update_contact = tch@sugarlabs.org
+developer_name = Sugar Labs Community
+developer_id = org.sugarlabs
+screenshots = https://raw.githubusercontent.com/sugarlabs/fototoon-activity/master/screenshots/en/1.png https://raw.githubusercontent.com/sugarlabs/fototoon-activity/master/screenshots/en/2.png https://raw.githubusercontent.com/sugarlabs/fototoon-activity/master/screenshots/en/3.png
